packages/pn-personafisica-webapp/src/App.tsx

```diff
@@ -165,7 +172,7 @@ const App = () => {
       icon: MailOutlineIcon,
       route: routes.NOTIFICHE,
       children: sideMenuDelegators,
-      notSelectable: sideMenuDelegators && sideMenuDelegators.length > 0,
+      notSelectable: sideMenuDelegators && sideMenuDelegators.length > 1,
```

Comment:
Prima se si avevano deleganti (length > 0) la voce di menù "Notifiche non era selezionabile" ora invece risulta selezionabile se si hanno 0 o 1 deleganti.
Sono cambiate le specifiche a riguardo?

Reply:
Le specifiche non sono certo cambiate.
Ma per l'altro, il caso sideMenuDelegators.length === 1 non si può dare, se hai n deleganti allora sideMenuDelegators avrà (n+1) elementi, il primo è "Le mie notifiche".
Questo l'ho cambiato per testare il controllo dell'API che ti dà i deleganti, in quel caso avevo pensato a aggiungere un'unico elemento in sideMenuDelegators che facesse il reload.
Ma visto che ho creato delle confusioni, restituisco il 0.